frontend/readme.md

```diff
@@ -24,7 +24,7 @@ See [Vite Configuration Reference](https://vitejs.dev/config/).
 ## Project Setup
 
 ```sh
-npm install
+npm run install-frontend
```

Comment:
Hm... I didn't notice frontend has two readme before.
I think we could delete this one and just keep the first README.md

Reply:
Please remove this readme.

Reply:
Checked with Ayobami on this readme duplication.
It is strange that I don't see that on his system. (Maybe something to do with same name files but UPPER case and lower case file names issue on Windows.)
I will resolve this one and delete it in other PR since Ayobami cannot see it.